app/forms/update_user_email_form.rb

```diff
@@ -15,7 +15,7 @@ def initialize(user)
   end
 
   def submit(params)
-    email = params[:email].downcase
+    email = params[:email].downcase.strip
```

Comment:
Yep. Note the addition of the extra whitespace in the email in this spec: spec/forms/update_user_email_form_spec.rb